### PR TITLE
Add Traccar icon

### PR DESCRIPTION
**Pull request by CodeShakingSheep**
image

**Issue:** closes https://github.com/simple-icons/simple-icons/issues/10888

**Popularity metric:**

The Similarweb rank is 330,066. See https://www.similarweb.com/website/traccar.org

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

I downloaded the header logo from the official website. Then, I converted the circle to a path using https://complexdan.com/svg-circleellipse-to-path-converter/ . I also applied transformations using this [Inkscape plugin](https://github.com/Klowner/inkscape-applytransforms). Then combined paths and optimized it.

### CODE REVIEW

**Comment by uncenter** [comment]
FYI you can use **Path** > **Object to Path** in Inkscape to convert circles (shapes in general) to paths instead of using that external tool.

File: icons/traccar.svg
Comment (uncenter) [inline icons/traccar.svg:1]: Screenshot 2024-05-06 at 16 29 55 (Inkscape)

I'm seeing slight differences on this semicircle in the bottom left. Could be issues with precision when optimizing.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/traccar.svg
Comment (CodeShakingSheep) [reply icons/traccar.svg:1]: Hm, yes likely due to precision. I increased the precision value to 4 and re-optimized the svg. Is it good now?

Comment (uncenter) [reply icons/traccar.svg:1]: Looks great!

**Review by uncenter — APPROVED** [review]

**Pull request merged**